versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.23"
+      version = ">= 5.30"
```

Comment:
we will wait until [v5.31.0](https://github.com/hashicorp/terraform-provider-aws/pull/34140) is released before adding support

Reply:
Ok, let's wait

Reply:
@bryantbiggs 
https://github.com/hashicorp/terraform-provider-aws/releases/tag/v5.31.0 is out.
do we keep `version = ">= 5.30"` of we should bump to `version = ">= 5.31"` ?

Reply:
5.31 please

Reply:
All done !